go/private/library.bzl

```diff
@@ -110,26 +97,23 @@ def emit_library_actions(ctx, srcs, deps, cgo_object, library, want_coverage, im
   for d in dep_runfiles:
     runfiles = runfiles.merge(d)
 
-  #TODO: for now, we return transformed sources, this will change
   transformed = dict_of(source)
   transformed["go"] = go_srcs
 
   return [
       GoLibrary(
-          library = out_lib,
-          race = race_lib,
-          searchpath = searchpath,
-          searchpath_race = searchpath_race,
-          srcs = join_srcs(struct(**transformed)),
-          importpath = importpath,
-          direct_deps = deps,
-          transitive_cgo_deps = transitive_cgo_deps,
-          transitive_go_libraries = transitive_go_library_deps + [out_lib],
-          transitive_go_libraries_race = transitive_go_library_deps_race + [race_lib],
-          transitive_go_library_paths = transitive_go_library_paths,
-          transitive_go_library_paths_race = transitive_go_library_paths_race,
-          gc_goopts = gc_goopts,
-          runfiles = runfiles,
+          importpath = importpath, # The import path for this library
```

Comment:
I plan to fully document the providers when I am finished changing them, as they are part of the stable api, but the documentation needs to go in def.bzl where they are aliased for public consumption, and I am not sure how to document them cleanly, I think I will have to just make up a syntax, as there is no prior art to work from.